### PR TITLE
[IMP] event_email_reminder: Add the possibility to select recipients

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by rafaelbn** [comment]
@pedrobaeza travis fails

```
======================================================================
2018-03-06 13:05:49,197 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: FAIL: test_cron_run_partners (openerp.addons.event_email_reminder.tests.test_event_event.TestEventEmailReminder)
2018-03-06 13:05:49,197 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: Traceback (most recent call last):
2018-03-06 13:05:49,197 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: `   File "/home/travis/build/OCA/event/event_email_reminder/tests/test_event_event.py", line 112, in test_cron_run_partners
2018-03-06 13:05:49,197 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: `     self.assertEqual(len(mails_to_send), 1)
2018-03-06 13:05:49,198 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: ` AssertionError: 0 != 1
2018-03-06 13:05:49,198 22349 INFO openerp_test openerp.addons.event_email_reminder.tests.test_event_event: Ran 5 tests in 6.964s
2018-03-06 13:05:49,198 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: FAILED
2018-03-06 13:05:49,198 22349 INFO openerp_test openerp.addons.event_email_reminder.tests.test_event_event:  (failures=1)
2018-03-06 13:05:49,198 22349 INFO openerp_test openerp.modules.module: openerp.addons.event_email_reminder.tests.test_event_event tested in 6.97s, 5259 queries
2018-03-06 13:05:49,198 22349 ERROR openerp_test openerp.modules.module: Module event_email_reminder: 1 failures, 0 errors
2018-03-06 13:05:49,370 22349 INFO openerp_test openerp.modules.module: module event_registration_cancel_reason: creating or updating database tables
```

**Comment by pedrobaeza** [comment]
Yeah, I need to check

**Comment by pedrobaeza** [comment]
@rafaelbn fixed

**Comment by rafaelbn** [comment]
@pedrobaeza travis fails

```
======================================================================
2018-03-06 13:05:49,197 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: FAIL: test_cron_run_partners (openerp.addons.event_email_reminder.tests.test_event_event.TestEventEmailReminder)
2018-03-06 13:05:49,197 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: Traceback (most recent call last):
2018-03-06 13:05:49,197 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: `   File "/home/travis/build/OCA/event/event_email_reminder/tests/test_event_event.py", line 112, in test_cron_run_partners
2018-03-06 13:05:49,197 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: `     self.assertEqual(len(mails_to_send), 1)
2018-03-06 13:05:49,198 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: ` AssertionError: 0 != 1
2018-03-06 13:05:49,198 22349 INFO openerp_test openerp.addons.event_email_reminder.tests.test_event_event: Ran 5 tests in 6.964s
2018-03-06 13:05:49,198 22349 ERROR openerp_test openerp.addons.event_email_reminder.tests.test_event_event: FAILED
2018-03-06 13:05:49,198 22349 INFO openerp_test openerp.addons.event_email_reminder.tests.test_event_event:  (failures=1)
2018-03-06 13:05:49,198 22349 INFO openerp_test openerp.modules.module: openerp.addons.event_email_reminder.tests.test_event_event tested in 6.97s, 5259 queries
2018-03-06 13:05:49,198 22349 ERROR openerp_test openerp.modules.module: Module event_email_reminder: 1 failures, 0 errors
2018-03-06 13:05:49,370 22349 INFO openerp_test openerp.modules.module: module event_registration_cancel_reason: creating or updating database tables
```

**Comment by rafaelbn** [comment]
Thanks @pedrobaeza ! do you know why runbot is unavailable?

**Comment by pedrobaeza** [comment]
Probably a glitch. Any way, just check code. It's very straight-forward

**Review by rafaelbn — APPROVED** [review]
Thanks

File: event_email_reminder/README.rst [hunk 1]
```diff
@@ -45,8 +49,10 @@ Images
 Contributors
 ------------
 
-* Sergio Teruel <sergio.teruel@tecnativa.com>
-* Vicent Cubells <vicent.cubells@tecnativa.com>
+* Tecnativa <https://www.tecnativa.com>:
+  * Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (cubells) [inline event_email_reminder/README.rst]: It's better for readability to add a blank line after Tecnativa line.

File: event_email_reminder/README.rst [hunk 2]
```diff
@@ -45,8 +49,10 @@ Images
...
+  * Vicent Cubells <vicent.cubells@tecnativa.com>
+  * Pedro M. Baeza
```
Comment (cubells) [inline event_email_reminder/README.rst]: Your email is missing.

**Review by cubells — APPROVED** [review]
Minor changes but module works.

:+1: 
LGTM 

Code review and test.

File: event_email_reminder/README.rst (hunk 2 above)
Comment (pedrobaeza) [reply event_email_reminder/README.rst]: I don't want to put anymore my mail, hehe

File: event_email_reminder/models/event.py [hunk 3]
```diff
@@ -10,17 +11,28 @@
...
     _inherit = 'event.event'
 
+    @api.model
+    def _send_event_template(self, events, template, partner_ids):
+        ctx = self.env.context.copy()
+        ctx.update({
+            'events': events.sorted(lambda x: x.date_begin),
+        })
+        for partner_id in partner_ids:
+            template.with_context(ctx).send_mail(partner_id)
```
Comment (chienandalu) [inline event_email_reminder/models/event.py]: A `lang` should be provided in context so the email is sent in the proper language:
```python
            ctx.update({
                'lang': partner_id.lang,
            })
```

Comment (pedrobaeza) [reply event_email_reminder/models/event.py]: Please make yourself the change for finishing this.

Comment (chienandalu) [reply event_email_reminder/models/event.py]: Done, I guess all commits can be rebased also, isn't it?

**Comment by pedrobaeza** [comment]
Yes

**Review by chienandalu — APPROVED** [review]
Fixed

File: event_email_reminder/models/event.py [hunk 4]
```diff
@@ -10,17 +11,31 @@
...
 
+    @api.model
+    def _send_event_template(self, events, template, partner_ids):
+        ctx = self.env.context.copy()
+        ctx.update({
+            'events': events.sorted(lambda x: x.date_begin),
+        })
+        for partner_id in partner_ids:
+            ctx.update({
+                'lang': self.env['res.partner'].browse(partner_id).lang,
```
Comment (pedrobaeza) [inline event_email_reminder/models/event.py]: This is still not working for event names, as you are putting the context for the template, not for the event itself.

**Comment by chienandalu** [comment]
@pedrobaeza Right, fixed

**Review by feketemihai — APPROVED** [review]

**Pull request merged**